ksql-engine/src/main/java/io/confluent/ksql/function/udf/json/ArrayContainsKudf.java

```diff
@@ -16,37 +16,34 @@
 
 package io.confluent.ksql.function.udf.json;
 
+import static com.fasterxml.jackson.core.JsonFactory.Feature.CANONICALIZE_FIELD_NAMES;
```

Comment:
This changes are inline with 'GoogleStyle', which we're all supposed to be using, but probably aren't. (More to come on this).